package.json

```diff
@@ -47,6 +47,7 @@
     "crypto-js": "^4.0.0",
     "fastestsmallesttextencoderdecoder": "^1.0.22",
     "mqtt": "^4.2.8",
+    "tar": "^6.1.11",
```

Comment:
We saw customer complaining of the deprecated dependencies too, are those tackled already? 
https://github.com/aws/aws-sdk-js-v3/issues/2750

Reply:
I saw those warnings as well, don't know how to solve them... `npm update` and `npm audit fix` is not helping... But, we can probably tackle them after